extensions/docker-desktop/apps/clustermgr/pkg/internal/docker/docker_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package docker
+
+import (
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/stretchr/testify/mock"
+	"github.com/vmware-tanzu/community-edition/extensions/docker-desktop/pkg/internal/docker/mocks"
```

Comment:
Will need a line break here to separate the third party imports from this local import to make the linter happy.

Reply:
/Ack